langgraph-sls-fastapi-rag/tests/integration/test_rag_graph.py

```
@@ -0,0 +1,48 @@
+    checkpoints = list(graph.checkpointer.list(config=config))
+    state_history = list(graph.get_state_history(config))
+    checkpoint_before_respond = checkpoints[1][1]['id']
+    config = RunnableConfig(
+        debug=True,
+        configurable={"user_id": user_id, "retriever_provider": "pinecone",
+                      "embedding_model": "openai/text-embedding-3-small",
+                      "response_model": "openai/gpt-4o-mini",
+                      "thread_id": "thread-1",
+                      "checkpoint_id": checkpoint_before_respond}
+    )
+    res = await graph.ainvoke(
```

Ruff (0.8.2)

Local variable `res` is assigned to but never used

Remove assignment to unused variable `res`

(F841)